documentation-website/Writerside/topics/Deep-Dive-into-DAO.md

```diff
@@ -173,6 +173,32 @@ class User(id: EntityID<Int>): IntEntity(id) {
 }
 ```
 
+### Ordered reference
+
+You can also define the order in which referenced entities appear:
+
+```kotlin
+class User(id: EntityID<Int>) : IntEntity(id) {
+    companion object : IntEntityClass<User>(Users)
+
+    ...
+    val ratings by UserRating referrersOn UserRatings.user orderBy UserRatings.value
+    ...
+}
+```
+
+In a more complex scenario, you can specify multiple columns along with the corresponding sort order for each:
+
+```kotlin
+class User(id: EntityID<Int>) : IntEntity(id) {
+    companion object : IntEntityClass<User>(Users)
+
+    ...
+    val ratings by UserRating referrersOn UserRatings.user orderBy listOf(UserRatings.value to DESC, UserRatings.id to ASC)
+    ...
```

Comment:
The fully qualified `SortOrder.DESC` etc should be used here since it's a sample (assuming it's only possible to use `DESC` if there's been an import).